vcpkg/maintainers/registries.md

```diff
@@ -1,281 +1,289 @@
 
-```powershell
-> git commit --amend
-```
+*Top-level fields*
 
-then share away!
+The top-level object in a `baseine.json` file is a dictionary, each key in this
+dictionary is a _named baseline_. Baselines should contain mappings of all the
+ports in the registry to their baseline version.
 
-### Filesystem registries
+| Name           | Type           | Description |
+| -------------- | -------------- | ----------- |
+| Named baseline | BaselineObject | Additional baselines. The field name corresponds to the baseline name. |
+
+*BaselineObject*
 
-As you're following along with this documentation, it may be helpful to have a working example to refer to. We've written one and put it here:
+The baseline object is a dictionary, with each key corresponding to a port name
+in the registry and its value being the latest version of the port.
 
-[Example filesystem registry](https://github.com/vcpkg/example-filesystem-registry).
+| Name      | Type                  | Description |
+| --------- | --------------------- | ----------- |
+| Port name | BaselineVersionObject | A mapping of a port name to its latest version |
 
-All filesystem registries must have a `versions/baseline.json` file. This file contains the set of "latest versions" for a certain version of the registry. It is laid out as a top-level object containing a map from version name to "baseline objects", which map port names to the version which is considered "latest" for that version of the registry.
+*BaselineVersionObject*
 
-Filesystem registries need to decide on a versioning scheme. Unlike git registries, which have the implicit versioning scheme of refs, filesystem registries can't rely on the version control system here. One possible option is to do a daily release, and have your "versions" be dates.
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `baseline` | string | A string corresponding to the latest available version of the port in the registry. |
+| `port-version` | integer | An integer corresponding to the latest port version of the port in the registry |
 
-> [!WARNING]
-> A baseline must not be modified once published. If you want to change or update versions, you need to create a new baseline in the `baseline.json` file.
+The layout of the baseline file in a filesystem registry is the same as for [Git
+registries](#git-baseline). The only difference being that filesystems
+don't require a `default` baseline.
 
-Here's an example of a valid `baseline.json`, for a registry that has decided upon dates for their versions:
+#### Example of a `baseline.json` file in a Git registry
 
 ```json
 {
-  "2021-04-16": {
-    "kitten": {
-      "baseline": "2.6.2",
-      "port-version": 0
-    },
-    "port-b": {
-      "baseline": "19.00",
-      "port-version": 2
-    }
-  },
-  "2021-04-15": {
-    "kitten": {
-      "baseline": "2.6.2",
-      "port-version": 0
-    },
-    "port-b": {
-      "baseline": "19.00",
+  "my-baseline": {
```

Comment:
I think this was better before when `my-baseline` was clearly a baseline revision rather than `my-baseline`.